docs/spelling_wordlist.txt

```diff
@@ -57,12 +57,12 @@ fallback
 Fernet
 fernet
 FIPS
+geenration
```

Comment:
Why was this added?

Reply:
In a (non-sensical) attempt to avoid future spelling errors, but the list of misspellings applies only to the docs.